public/server/db/pitr.json

```
@@ -0,0 +1,3 @@
+{
+    "definition": "Parent In The Room",
```

Since `definition` is the only key here, this comma doesn't need to be at the end as it'll cause errors 
```suggestion
    "definition": "Parent In The Room"
```